cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -1337,6 +1337,11 @@ func (snr *sovereignNodeRunner) CreateManagedProcessComponents(
 		return nil, err
 	}
 
+	outgoingOperationsHasher, err := hasherFactory.NewHasher("sha256")
```

Comment:
Hasher type should be a configurable paramter found and loaded from `sovereignConfig.toml`